Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -606,8 +609,6 @@ - (void)scrollToDefaultPosition
     } else {
         [self scrollToBottomAnimated:NO];
     }
-
-    [self updateLastVisibleTimestamp];
```

Comment:
We need a thorough test plan around this, but think we were updating the "last visible" state way too often.  It should be sufficient to update it:

* On view will appear.
* When collection view contents are changed with `performBatchUpdates`.
* When collection view scroll state changes.